permissions/src/androidTest/java/com/google/accompanist/permissions/DummyTests.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.permissions
+
+import org.junit.Test
+
+/**
+ * Dummy test to avoid the "No tests found error" when running in Build.VERSION.SDK_INT < 23
+ */
+class DummyTests {
+
+    @Test
+    fun dummyTestToAvoidNoTestsFoundErrorInAPI22AndBelow() = Unit
```

Comment:
Can probably remove this one

Reply:
I'd rather leave it there in case we need to remove the other dummy tests. This is a reminder that we cannot remove it :D